README.md

```diff
@@ -830,7 +830,7 @@ Write code on a whiteboard or paper, not a computer. Test with some sample input
         - [ ] is_empty() - returns true if heap contains no elements
         - [ ] extract_max - returns the max item, removing it
         - [ ] sift_down - needed for extract_max
-        - [ ] remove(i) - removes item at index x
+        - [ ] remove(x) - removes item at index x
```

Comment:
@jwasham ah, this has to be noted.

I did not mean to contain that commit, but it is useful.
It is just typo, should be make an issue for it?

and in my opinion, this looks like more accurate:
```
- - [ ] remove(x) - removes item at index x
+ - [ ] remove(i) - removes item at index i
```

the `i` for index, not the `x`, what do you think about this?

Reply:
Making them match is logical. Feel free to open an issue.